src/otx/core/data/module.py

```diff
@@ -40,18 +40,25 @@ def __init__(self, task: OTXTaskType, config: DataModuleConfig) -> None:
             format=self.config.data_format,
         )
 
+        available_name_mapping = {
+            self.config.train_subset_name: "train",
+            self.config.val_subset_name: "val",
+            self.config.test_subset_name: "test",
+        }
```

Comment:
This may introduce the confusions between overall code. How about defining "train", "val", "test" as constants such as `SUBSET_NAME_TRAIN`, ...?

Reply:
Let me clean up these things prettier.

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/2685/commits/a2ed769efc9007cd6eed669ed6c29a7fa11881b1 tried some improvements for cleaning. However, `OTXDataModule` is under development and we have a chance to improve its interface further in the future.